### PR TITLE
Update retry queue processing

### PR DESCRIPTION
**Pull request by reachjainrahul**
## Description
Retry queue was processed every second and the retry count was 15. So, for a failed cloud operation, it was retrying very fast and too many number of times.

## Changes
With this PR, we are reducing the number of retry to 3 and increased the retry interval to 20 sec. So essentially a failed cloud response will be retried for 1 minute with 20 sec gap.

### CODE REVIEW

File: pkg/controllers/networkpolicy/pendingitem.go [hunk 1]
```diff
@@ -24,6 +25,11 @@ import (
 	antreanetworking "antrea.io/antrea/pkg/apis/controlplane/v1beta2"
 )
 
+var (
+	retryInterval int64 = 20
```
[the remark this answers is not in the document]
Comment (reachjainrahul) [reply pkg/controllers/networkpolicy/pendingitem.go:29]: it is required.

**Review by shenmo3 — APPROVED** [review]
LGTM

**Pull request merged**